### PR TITLE
[chore][readme] Clarify purpose of Slack and video calls

### LINKED ISSUE #7085: [Proposal] Add an additional APAC collector/agent SIG
Apologies if I have opened this in the wrong spot, please feel free move it into the correct place.

I am looking for an expression of interesting in running an additional meeting for those working in APAC and wanting to actively contribute back to the project.

Looking at the otel calendar,  4:00pm Pacific (Taken from the otel Instrumetation APAC SIG) works well for me. 

Or roughly -22:00 UTC through to +08:00 UTC (negative being the previous day respective to UTC)

### PR DESCRIPTION
**Pull request by jpkrohling**
This PR implements the proposal at #7085 while addressing the following concerns:

- makes it explicit that ad-hoc meetings are possible
- sets a timeline for the split of ad-hoc calls
- sets the expectation that people don't have to join the calls to be heard
- and the expectation that everything that was decided is tracked via github

Closes #7085

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,38 @@ Objectives:
 - Extensible: Customizable without touching the core code.
 - Unified: Single codebase, deployable as an agent or collector with support for traces, metrics and logs.
 
+## Community
```
Comment (mx-psi) [inline README.md:62]: Should this be in CONTRIBUTING.md instead?

**Review by mx-psi — COMMENTED** [review]
Content looks good to me, just a question of where to put this

File: README.md (hunk 1 above)
Comment (jpkrohling) [reply README.md:62]: I considered that, but I'd like to keep this open to the community in general, especially the mention of the Slack channel.

**Review by axw — COMMENTED** [review]
Love it!

Just one comment:

> - look for a sponsor for a proposed component
> - get attention to a specific pull-request

Should these be considered exceptions to the norm? Seems to me these should typically be doable asynchronously.

I understand that meetings have value for synchronous discussions, I just think it would be preferable to go with an async-first approach for as much as possible, in as large and geographically-distributed set of contributors as this project has.

**Comment by jpkrohling** [comment]
> Should these be considered exceptions to the norm? Seems to me these should typically be doable asynchronously.

I'll clarify that. While most of our PRs are reviewed without an explicit request at a synchronous call, I think most of the new components have difficulties finding sponsors, so proponents end up joining the SIG calls to request sponsors. The steps I usually recommend for new components are to open an issue, ask on Slack, and ask on the SIG call, each step a week or so before the next one.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,38 @@ Objectives:
...
+
+Between 11 July 2024 and 09 January 2025, we'll have our video calls rotating between three time slots, in order to
+allow everyone to join at least once every three meetings:
+
+- [00:00 UTC](https://dateful.com/convert/utc?t=0000)
+- [12:00 UTC](https://dateful.com/convert/utc?t=1200)
+- [16:00 UTC](https://dateful.com/convert/utc?t=1600)
+
+Contributors to the project are also welcome to have ad-hoc meetings for synchronous discussions about specific points.
+Here is a list of tools that can be used for this:
```
Comment (codeboten) [inline README.md:81]: I'd suggest the list of tools here is unnecessary and would rather call out how one setup an ad-hoc meeting (ie. post a note in slack that says hey i want to meet to chat about X)

**Review by codeboten — COMMENTED** [review]
Thanks @jpkrohling for driving this forward

File: README.md [hunk 3]
```diff
@@ -59,6 +59,34 @@ Objectives:
...
+
+Between 11 July 2024 and 09 January 2025, we'll have our video calls rotating between three time slots, in order to
+allow everyone to join at least once every three meetings:
+
+- [00:00 UTC](https://dateful.com/convert/utc?t=0000)
+- [12:00 UTC](https://dateful.com/convert/utc?t=1200)
+- [16:00 UTC](https://dateful.com/convert/utc?t=1600)
+
+Contributors to the project are also welcome to have ad-hoc meetings for synchronous discussions about specific points.
+Post a note on #otel-collector inviting others, specifying the topic to be discussed. Unless there are strong reasons to
```
Comment (codeboten) [inline README.md:81]:
```suggestion
Post a note in #otel-collector on Slack inviting others, specifying the topic to be discussed. Unless there are strong reasons to
```

**Review by codeboten — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

**Review by songy23 — APPROVED** [review]

**Review by crobert-1 — APPROVED** [review]

**Pull request merged**